src/inject/index.js

```diff
@@ -110,7 +110,7 @@ function getNewContent(target, collection, opt) {
       var pluralState = filesCount > 1 ? 's' : '';
       log(cyan(filesCount) + ' file' + pluralState + ' into ' + magenta(target.relative) + '.');
     } else {
-      log('Nothing to inject into ' + magenta(target.relative) + '.');
+      error('Nothing to inject into ' + magenta(target.relative) + '.');
```

Comment:
You'll need a `throw` here as well, otherwise nothing will happen, it won't even log anything...

Reply:
Done